includes/admin/views/html-admin-page-addons.php

```diff
@@ -28,8 +28,15 @@
 			</div>
 		<?php endif; ?>
 		<?php if ( 'featured' !== $current_section && $addons = WC_Admin_Addons::get_section_data( $current_section ) ) : ?>
+			<div class="addons-section-blocks">
+				<?php
+					if( $addons['blocks'] ) {
+						WC_Admin_Addons::output_sections( $addons['blocks'] );
+					}
+				?>
```

Comment:
Please use the PHP alternative syntax for control structures when mixing HTML and PHP. ([#](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style))

Should the `.addons-section-blocks` div be conditionally rendered?